src/MediaRecorder.ts

```diff
@@ -48,9 +53,13 @@ export default class MediaRecorder {
      * @param filePath The path where we'll save the file
      */
     start(filePath) {
-        // TODO: start recording, and save the path
-        // use react-native-fs for filePath
-        throw Error('Unimplemented');
+        // TODO: use react-native-fs for filePath
```

Comment:
I can't remember what this was for 🤷 I'm trying to remember if this was for the calling app to use something like react-native-fs to pass a proper filename. Also unsure if we would want to ensure any directories are made ahead of time or just make the calling app responsible for that 🤔

Reply:
Regardless and maybe we need to sync back up, but lets make sure whatever is decided gets documented somewhere and likely a ticket made (potentially just to add some info the method comment)

Reply:
Added a ticket with our decision summarized: https://blinemedical.atlassian.net/browse/APPS-600